csharp-basic-topics/StringBuilderType/Tests/Tests.cs

```diff
@@ -0,0 +1,220 @@
+namespace Tests;
+
+using StringBuilderType;
+using System;
+using Xunit;
+
+public class Tests
+{
+    public static readonly StringBuilderMethods stringBuilderMethods = new StringBuilderMethods();
+
```

Comment:
When naming tests, the "given" part is optional, and the "when" part is mandatory. Please either use both or just "when". Also the "then" part begins with the word "Then". Check out the docs for some examples.